sections/main-search.liquid

```diff
@@ -111,10 +111,11 @@
                                   {%- if item.image.src.width >= 360 -%}{{ item.image.src | img_url: '360x' }} 268w,{%- endif -%}
                                   {%- if item.image.src.width >= 533 -%}{{ item.image.src | img_url: '533x' }} 357w,{%- endif -%}
                                   {%- if item.image.src.width >= 720 -%}{{ item.image.src | img_url: '720x' }} 536w,{%- endif -%}
-                                  {%- if item.image.src.width >= 940 -%}{{ item.image.src | img_url: '940x' }} 714w{%- endif -%}"
-                              src="{{ item.image.src | img_url: '533x' }}"
+                                  {%- if item.image.src.width >= 940 -%}{{ item.image.src | img_url: '940x' }} 714w{%- endif -%}
+                                  {%- if item.image.src.width >= 1066 -%}{{ item.image.src | img_url: '1066x' }} 1066w{%- endif -%}"
```

Comment:
The `1066` comes from a 3x at `749px`.  Rather than base it on devices, which change over time, I've based it on the actual breakpoints. The images are 355, so 355 * 3 = 1065 (not 1066, so that was my bad).